config.toml

```diff
@@ -55,6 +55,11 @@ anchor = "smart"
     name = "Slack"
     weight = 50
     url = "https://app.slack.com/client/TUTSYURT3/C0103QJMA84"
+[[menu.main]]
+    name = "Donate"
+    weight = 50
```

Comment:
I'd set weight to 60, just to offset it from the slack link.

Reply:
That's a fair point. Since it's not getting merged rn I'll push that in when I add the link